### PR TITLE
Feat: Our values

### PR DESCRIPTION
**Pull request by MiraMC149**
This is the values banner you requested

### CODE REVIEW

File: src/components/Consume.jsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
+import React from "react";
+
+function Consume() {
+  return (
+    <div>
+      <svg
```
Comment (osamaakb) [inline src/components/Consume.jsx:6]: please move `SVGs` to a separate file

File: src/components/Donate.jsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        id="Layer_1"
+        x="0px"
+        y="0px"
+        width="121px"
+        height="91px"
+        viewBox="0 0 121 91"
+        enablebackground="new 0 0 121 91"
+        xmlSpace="preserve"
+      >
+        {" "}
```
Comment (osamaakb) [inline src/components/Donate.jsx:17]: remove this `{" "}`

File: src/components/Donate.jsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
+import React from "react";
+
+function Donate() {
+  return (
+    <div className="-ml-20">
+      <svg
```
Comment (osamaakb) [inline src/components/Donate.jsx:6]: same here

File: src/components/Planet.jsx [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+        id="Layer_1"
+        x="0px"
+        y="0px"
+        width="110px"
+        height="81px"
+        viewBox="0 0 110 81"
+        enablebackground="new 0 0 110 81"
+        xmlSpace="preserve"
+      >
+        {" "}
```
Comment (osamaakb) [inline src/components/Planet.jsx:17]: same here and everywhere else

File: src/components/Awarness.jsx [hunk 5]
```diff
@@ -0,0 +1,173 @@
+import React from "react";
+
+function Awarness() {
```
Comment (osamaakb) [inline src/components/Awarness.jsx:3]: is this an icon or a component?

**Review by osamaakb — CHANGES_REQUESTED** [review]
**Please**, stick to the technical manual and follow the instructions there regarding the folders structure

**Comment by osamaakb** [comment]
same here, you have some conflicts to fix

**Review by osamaakb — CHANGES_REQUESTED** [review]
otherwise looks good, just add more padding at the bottom, it doesn't match the one on top

image

File: src/components/OurValues/OurValues.jsx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+import QualityIcon from "../../Icons/QualityIcon";
+import AwarnessIcon from "../../Icons/AwarnessIcon";
+
+function OurValues() {
+  return (
+    <div className="flex flex-row bg-white w-full pt-4 pb-10">
+      <p className="font-semibold text-3xl tracking-tighter text-bric ml-20 mt-10">
+        Our Values
+      </p>
+      <div className="flex flex-col mr-60 mt-20 ">
```
Comment (osamaakb) [inline src/components/OurValues/OurValues.jsx:15]: you shouldn't add margin-right to each item to make them look like a grid, you can achieve that using flex-box wrap and add space between flex item.

read here
https://css-tricks.com/dont-overthink-flexbox-grids/

and here
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Relationship_of_Grid_Layout

**Review by osamaakb — CHANGES_REQUESTED** [review]

**Comment by MiraMC149** [comment]
I finally managed to resolve the issues

**Review by osamaakb — APPROVED** [review]
Good job @MiraMC149

**Pull request merged**